docs/ui/components/Table.tsx

```diff
@@ -0,0 +1,70 @@
+import { css } from '@emotion/react';
+import { theme, borderRadius } from '@expo/styleguide';
+import React from 'react';
+
+type TableProps = {
+  headers: string[];
+  children: React.ReactNode;
+};
+
+export const Table: React.FC<TableProps> = ({ children, headers }) => (
+  <div css={tableWrapperStyle}>
+    <table css={tableStyle}>
+      {headers && headers.length ? (
```

Comment:
I'm not sure if adding a `headers = []` default value would help, but also wanted to add that as alternative for this check 😄